src/app/components/details/album_header.ui

```diff
@@ -95,6 +95,19 @@
         </child>
       </object>
     </child>
+    <child>
```

Comment:
I think we should have some box around the buttons if we're goign to have more than one -- see how it looks on mobile